scib/metrics/metrics.py

```diff
@@ -493,25 +485,36 @@ def metrics(
         )
     else:
         trajectory_score = np.nan
+    
+    if moransi_: 
+        print("Moran's I score...")
+        moransi_score = morans_i(
+            adata, 
+            adata_int, 
+            batch_key=batch_key
+        )
```

Comment:
looks like your auto-formatting didn't strike here... inconsistency with comma after last parameter or not? ^^

Reply:
This is corrected on #304